src/upgrade-compatibility-tool/run.md

```diff
@@ -137,4 +137,4 @@ Regarding the GraphQL schema compatibility comparison, the output would be very
  *   [WARNING] OPTIONAL_INPUT_FIELD_ADDED: An optional field sku on input type ProductAttributeSortInput was added.
 ```
 
-See [Developer information]({{site.baseurl}}/safe-upgrade-tool/developer.html) for more information.
+See [Developer information]({{site.baseurl}}/upgrade-compatibility-tool/developer.html) for more information.
```

Comment:
Repo name has not changed yet

Reply:
this refers to the folder in the docs, nothing to do with the readme or existing repo